libsplinter/src/node_registry/error.rs

```diff
@@ -75,6 +75,7 @@ pub enum InvalidNodeError {
     EmptyEndpoint,
     EmptyIdentity,
     EmptyDisplayName,
+    InvalidIdentity(String, String), // (identity, msg)
```

Comment:
Should this be more general? Like `InvalidUpdate`?

Reply:
This is general in the context of the `InvalidNodeError`. An `InvalidUpdate` variant would not be appropriate since this error type is used for non-update actions as well.